app/src/test/java/org/simple/clinic/summary/bloodsugar/BloodSugarSummaryViewEffectHandlerTest.kt

```diff
@@ -27,4 +30,14 @@ class BloodSugarSummaryViewEffectHandlerTest {
     //then
     testCase.assertOutgoingEvents(BloodSugarSummaryFetched(measurements))
   }
+
+  @Test
+  fun `when open blood sugar type selector effect is received then type selector sheet should be opened`() {
+    //when
+    testCase.dispatch(OpenBloodSugarTypeSelector)
+
+    //then
+    testCase.assertNoOutgoingEvents()
+    verify(uiActions).showBloodSugarTypeSelector()
```

Comment:
We should also use `verifyNoMoreActions(ui)` here to check that nothing else got triggered accidentally.

Reply:
Done